ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -194,14 +194,12 @@ extension PersonaListView: UITableViewDataSource {
                 guard let cell = dequeueReusableCell(withIdentifier: ActivityIndicatorCell.identifier, for: indexPath) as? ActivityIndicatorCell else {
                     return UITableViewCell()
                 }
-                cell.hideSystemSeparator()
                 return cell
             case .displayingSearchResults:
                 guard let cell = dequeueReusableCell(withIdentifier: CenteredLabelCell.identifier, for: indexPath) as? CenteredLabelCell else {
                     return UITableViewCell()
                 }
                 cell.setup(text: searchResultText)
-                cell.hideSystemSeparator()
```

Comment:
The `ActivityInidicatorCell` takes care of it in init, and the `ActionsCell` does it in layoutSubviews, but do we not need to push the system separator off screen anymore with the `CenteredLabelCell`?

Reply:
this is on purpose. because the personlistview the tableview itself hides the separator. so no behavioral changes.